src/Integration.Vsix/Commands/PackageCommandManager.cs

```diff
@@ -58,6 +61,7 @@ public PackageCommandManager(IMenuCommandService menuService)
 
             // Help menu buttons
             this.RegisterCommand(CommonGuids.HelpMenuCommandSet, (int)PackageCommandId.SonarLintHelpShowLogs, new ShowLogsCommand(outputWindowService));
+            this.RegisterCommand(CommonGuids.HelpMenuCommandSet, AboutCommand.AboutCommandId, new AboutCommand(browserService));
```

Comment:
It's different from the other commands, but I like it this way more. Should we discuss making that to be the way of defining the ids for future commands?

Reply:
Found a possible issue with this approach in the other PR

Reply:
Yes, we will do it this way now I changed it in my prs but I didnt change it in the already merged one as that should be a seperate pr :)

Reply:
Could you look at my comments on the other PR, I found an issue with this approach

Reply:
@georgii-borovinskikh-sonarsource can you link to it?